### PR TITLE
DEVX-477 some fixes and new APM integration with open telemetry

### PR DESCRIPTION
**Pull request by barbara79**
- [ ] Changeset added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -959,6 +959,16 @@ To migrate from the 1.x to the 2.x, there is a guideline below:
 
 To monitor and observe the SDK, see the official documentation [Observability](https://docs.commercetools.com/sdk/observability), there is a [Demo application](https://github.com/commercetools/commercetools-sdk-php-v2/tree/master/examples/symfony-app) which shows how to monitor the PHP SDK with New Relic and Datadog.
```
Comment (lojzatran) [inline README.md:960]: What about dynatrace and otel?

**Review by lojzatran — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ The SDK consists of the following projects:
 * `lib/commercetools-history/src`: Contains all generated models and request builders to communicate with the [Change History API](https://docs.commercetools.com/api/history/change-history).
 
 In addition, the SDK has the following directories:
-* `examples/symfony-app` : it shows how the PHP SDK can be used in a Symfony application for a Docker environment with NewRelic monitoring enabled. 
+* `examples/` : One Demo Symfony Application in a Dockerized environment for each APM (NewRelic, Datadog, Dynatrace and OpenTelemetry), which shows how the PHP SDK can be used from the user side.
```
Comment (VictorAvelar) [inline README.md:70]:
```suggestion
* `examples/` : One Dockerized Symfony app per APM (New Relic, Datadog, Dynatrace, OpenTelemetry) to demo PHP SDK usage. 
```

**Review by VictorAvelar — APPROVED** [review]
Nit: I believe this sentence can be simplified.

File: README.md (hunk 1 above)
Comment (barbara79) [reply README.md:960]: you are right I skipped this update, I am gonna do it immediately :D
Thank you!

**Pull request merged**